src/Chainweb/Pact/SPV.hs

```diff
@@ -77,6 +79,19 @@ import Pact.Types.PactValue
 import Pact.Types.Runtime
 import Pact.Types.SPV
 
+catchAndDisplaySPVError :: BlockHeader -> ExceptT Text IO a -> ExceptT Text IO a
+catchAndDisplaySPVError bh =
+  if CW.chainweb219Pact (_blockChainwebVersion bh) (_blockHeight bh)
+  then flip catch $ \case
+    SpvExceptionVerificationFailed m -> throwError ("spv verification failed: " <> m)
+    spvErr -> throwM spvErr
+  else id
```

Comment:
Ehhhh let's just eta expand if we're at the point where we're using `flip` and `id` to avoid passing arguments.

```suggestion
catchAndDisplaySPVError :: BlockHeader -> ExceptT Text IO a -> ExceptT Text IO a
catchAndDisplaySPVError bh action =
  if CW.chainweb219Pact (_blockChainwebVersion bh) (_blockHeight bh)
  then catch action $ \case
    SpvExceptionVerificationFailed m -> throwError ("spv verification failed: " <> m)
    spvErr -> throwM spvErr
  else action
```

Reply:
Are we supposed to edit this PR? Aren't these changes that belong to other PRs? I think, it may get confusing if we review and change the same PR in different places.